php/src/Core/Database/Connector.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace App\Core\Database;
+
+use Cassandra;
+use Cassandra\FutureRows;
+use Cassandra\Session;
+use Cassandra\SimpleStatement;
+
+class Connector
+{
+    /**
+     * @var Session
+     */
+    public $cluster;

```

Comment:
```suggestion
    public Session $cluster;
```

Reply:
Since I'm working on PHP7.1 it's not possible. Probably soon I'll have some answer about how to implement the PHP 8.x and then we can fix it.